crates/forge_analyzer/src/definitions.rs

```diff
@@ -469,12 +548,35 @@ struct ExportCollector<'cx> {
     default: Option<DefId>,
 }
 
+struct GlobalCollector<'cx> {
+    res: &'cx mut Environment,
+    module: ModId,
+    global_id: DefId,
+    secret_packages: Vec<PackageData>,
+    parent: Option<DefId>,
+}
+
 #[derive(Debug, Clone, Copy, PartialEq, Eq)]
 enum LowerStage {
     Collect,
     Create,
 }
 
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum IntrinsicName {
+    RequestConfluence,
+    RequestJira,
+    Other,
+}
+
+#[derive(Serialize, Deserialize, PartialEq, Debug, Clone, Eq)]
+pub struct PackageData {
+    pub package_name: String,
+    pub identifier: String,
```

Comment:
I think this should be the actual identifier that's exported. For `default` exports, this makes sense, because `export default` technically names the exported object or function "default". However, "star" should technically just be the actual function or object identifier, since the package itself doesn't really care if its being imported as a named or star import. For instance:

```javascript
import * as jwt from 'jsonwebtoken'
jwt.sign('foo','secret');

//can also be imported like this:
import { sign } from 'jsonwebtoken';
sign('foo', 'secret');
```
So the identifier should simply be 'sign'. By keeping track of the identifier there, we can also support objects like in `crypto-js` by renaming `function_name` to `method` and having it be an `Option<String>`. So if we wanted to support `AES.encrypt` (which is an object) in `crypto-js`, we would have:

```yaml
package_name: crypto-js
identifier: AES
method: encrypt
secret_position: 2
```